manuals/UserGuide/UserGuide-UI_Simulator-Visualizers.rst

```diff
@@ -595,3 +595,26 @@ The user can also input the slice manually.
    :align: center
 
    Preview for Matrix Visualizer
+   
+   
+Connectivity Edge Bundle Visualizer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+
+Shows structural connectivity coming in and out of a brain region by highlighting paths to other regions.
+
+.. figure:: screenshots/connectivity_edge_bundle.png
```

Comment:
Please:
- rename the new screenshots according to existing naming schema for the others
 visualizer_connectivity_edge_bundle.jpg and visualizer_perason_edge_bundle.jpg
- make the screen-shots JPG (like the others)
- cut from the image the browser edges